src/client/js/otp/config.js

```diff
@@ -324,6 +324,10 @@ otp.config.modes = {
     //TRANSLATORS: Travel by: mode of transport (Used in selection in Travel
     //Options widgets) (Park bicycle at Public transit station and take a
     //transit
```

Comment:
Translator comment here is wrong as it is copied from Bike and Ride, not from earlier Kiss and Ride.